### ISSUE TITLE
Can't perform "standing actions" when freefloating with momentum

### ISSUE THREAD

**Issue by AnonymousNow** [tgstation/tgstation#29770]
Trying to perform an action in zero-gravity while you have directional momentum that you are not causing yourself (eg. trying to uncuff yourself or apply a medical patch while floating through space unassisted by a jetpack or the like) fails, as if you are walking, running or moving of your own volition.

I don't think that this is a bug - more an oversight, perhaps. It seems illogical that a person who is, relatively speaking, still and concentrating on one task in zero-gravity would fail because they're drifting a little.

Discuss?

**Comment by Marflowerose** [comment]
This was brought up some time ago and was never really concluded. Regardless of if you should or should not be do those actions in space while moving, it's still currently inconsistent; You can jab yourself with a syringe to the eye but not apply a patch. I'm of the mind that you should be able to do all the actions while moving zero-g, provided that you're not spinning around your own axis many times per second where the actual g-forces from the spin would render you unconscious.

**Comment by Cyberboss** [comment]
This is a dupe

**Comment by Marflowerose** [comment]
Yeah, dupe of https://github.com/tgstation/tgstation/issues/10235 though apparently there was a fix? https://github.com/tgstation/tgstation/pull/14187

**Comment by XDTM** [comment]
It was fixed at some point, what brought it back?
